Src/MoneyFox.Windows.Business/BackgroundTaskManager.cs

```diff
@@ -2,28 +2,68 @@
 using System.Linq;
 using Windows.ApplicationModel.Background;
 using MoneyFox.Foundation.Interfaces;
+using System.Collections.ObjectModel;
+using MoneyFox.Foundation.DataModels;
+using MoneyFox.Foundation.Interfaces.Repositories;
+using MoneyFox.Business.Manager;
 
 namespace MoneyFox.Windows.Business
 {
     public class BackgroundTaskManager : IBackgroundTaskManager
     {
-        private const string TASK_NAMESPACE = "MoneyFox.Windows.Tasks";
+        private readonly ISettingsManager settingsManager;
+        private readonly ICategoryRepository categoryRepository;
 
+        private const string TASK_NAMESPACE = "MoneyFox.Windows.Tasks";
         private const string CLEAR_PAYMENT_TASK = "ClearPaymentTask";
         private const string RECURRING_PAYMENT_TASK = "RecurringPaymentTask";
         private const string SYNC_BACKUP_TASK = "SyncBackupTask";
 
+        public BackgroundTaskManager(ICategoryRepository categoryRepository, ISettingsManager settingsManager)
+        {
+            this.categoryRepository = categoryRepository;
+            this.settingsManager = settingsManager;
+        }
+
+        public bool firstUse
```

Comment:
as you mentioned, I guess this would make more sense in a assistance class than the background manager..
Also, is there a need for a publich property? If yes, the Name should be starting with a Upper case  "F" :)